packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_misc.spec.ts

```diff
@@ -7,8 +7,97 @@ context('multi-domain misc', { experimentalSessionSupport: true, experimentalMul
 
   it('verifies number of cy commands', () => {
     // @ts-ignore
-    expect(Object.keys(cy.commandFns).length).to.equal(86,
-      'The number of cy commands has changed. Please ensure any newly added commands are also tested in multi-domain.')
+    expect(Object.keys(cy.commandFns)).to.eql(
```

Comment:
```suggestion
    expect(Object.keys(cy.commandFns)).to.deep.equal(
```

Reply:
I think `eql` is the same as `deep.equal`. https://www.chaijs.com/api/bdd/#method_eql am I reading the docs wrong?

Reply:
I guess this is the difference: 

>The .deep.equal assertion is almost identical to .eql but with one difference: .deep.equal causes deep equality comparisons to also be used for any other assertions that follow in the chain.

Reply:
Yeah, I think using `.equal` and `.deep.equal` more clearly defines the intent but since there isn't a defined rule it's probably fine to use `.eql` as well.

Reply:
updated